pom.xml

```diff
@@ -179,7 +179,7 @@
         <!-- keep dependency properties sorted -->
         <dep.accumulo-hadoop.version>2.7.7-1</dep.accumulo-hadoop.version>
         <dep.accumulo.version>3.0.0</dep.accumulo.version>
-        <dep.airlift.version>247</dep.airlift.version>
+        <dep.airlift.version>248</dep.airlift.version>
```

Comment:
How come this works without updating discovery?

Reply:
Discovery was updated along with 247

Reply:
And this is 248